### PR TITLE
[RCIAM-82] Added config option that enables a CO Enrollment Flow to be hidden fr…

### PR DESCRIPTION
**Pull request by ioigoume**
# Feature
Added the configuration variable **hidden** in **Co Enrolment Flows** configuration page. By default the value is false/empty and all the configured Enrolment Flows will be displayed in `People->Enrol` page. If the Admin changes the value to true, then the Flow will be hidden.
This is very handy for **Sign Up** Flows or other flows that we want to hide from the users; since they have no meaning for them and yet they cause frustration.

The Admin will still be able to manage the flow via the configuration tab.

For this feature to work we need to add a column in table `cm_co_enrollment_flows`
SQL command:
```
alter table cm_co_enrollment_flows add column hidden boolean;
```

### CODE REVIEW

**Review by NicolasLiampotis — APPROVED** [review]
- Rename hide to hidden
- Update CHANGELOG.md

File: app/Config/Schema/schema.xml [hunk 1]
```diff
@@ -1332,6 +1332,7 @@
     <field name="redirect_on_finalize" type="C" size="1024" />
     <field name="return_url_whitelist" type="X" size="4000" />
     <field name="ignore_authoritative" type="L" />
+    <field name="hide" type="L" />
```
Comment (NicolasLiampotis) [inline app/Config/Schema/schema.xml]: hidden

Comment (ioigoume) [reply app/Config/Schema/schema.xml]: Done.

**Review by NicolasLiampotis — APPROVED** [review]
LGTM

**Pull request merged**